### ISSUE TITLE
WCAG 4.1.2 test 9 false positives

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#228]
[Test document 20](https://github.com/veraPDF/veraPDF-wcag-algs/files/9120168/test-document-20.pdf) fails WCAG 2.1 4.1.2 test 9 (_Missing Table elements_) on the entire document. There is a single occurrence of the error, but it points to all the content in the document.

This is an issue that was previously raised, resolved and closed, but apparently the table detection algorithm still has problems with some documents.

**Comment by bdoubrov** [comment]
We've implemented a partial fix that already improves the situation. Not yet ideal though.

**Comment by bdoubrov** [comment]
Current version still has a single false positive on incorrect Table dectection: two heading on page 18 are perfectly aligned and might indeed look as a 2 column table. I suggest to close this issue and open a new one for this remaining false positive, as it seems to be a different case.
